### ISSUE TITLE
[Bug] Not receiving items

### ISSUE THREAD

**Issue by DeusDrizzyy** [andrei1058/BedWars1058#298]
**Describe the bug:**
When the server restarts with a match in progress, players keep the match items instead of receiving items from config.yml (lobby-items) when connecting to the lobby.

**To Reproduce:**
1. Join a match and wait for it to boot.
2. After the match starts and you receive the standard items, in this example a wooden sword, just restart the server from the console.
3. Wait for the server to restart completely and enter the lobby.

**Expected behavior**
Items that the player has in their inventory must be removed, and the player will receive the default items that are in the configuration. (config.yml -> lobby-items)

**Screenshots**
- Hotbar - When joining the server for the first time: https://i.imgur.com/oH29Xb7.png
- Hotbar - After the match starts: https://i.imgur.com/JOQJ2x7.png
- Hotbar - Items from the hotbar of the player who is in the lobby after the server restarts: https://i.imgur.com/JOQJ2x7.png

**System:**
 - OS: Linux
 - Server software: Spigot
 - Server Version: 1.8.8
 - Plugin Version: 21.12.1

**Additional context:**
Bottom line: players inventory is not being cleared after server restart (when first joining server), this causes default lobby items not to be added to players.

**Comment by DeusDrizzyy** [comment]
Note: If the player is a spectator when the server restarts, in addition to not receiving items from the lobby, they will still have the invisibility effect when joining the server after the restart.
